Algorithms/Fenwick.cpp

```diff
@@ -0,0 +1,53 @@
+//
```

Comment:
Approving for now, but this should be slightly refactored to be more reusable/portable